### ISSUE TITLE
Uploads don't work

### ISSUE THREAD

**Issue by MarcS1975** [nextcloud/android#7764]
### Steps to reproduce
1. When I want to upload a file (such as an image) to Nextcloud via Android "sharing" function, a quick nextcloud message box pops up but the upload does not work. There is no file on the server. 
2. The upload via the Nextcloud app using the "upload file" function does work though. 
3. 

### Expected behaviour
- fille should be copied from app (like WhatsApp or Android gallery to Nextcloud server

### Actual behaviour
- popup of nextcloud message and close immediately. No file is uploaded. 

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:
10
Device model: 
Samsung s9
Stock or customized system:

Nextcloud app version:
3.14.1
Nextcloud server version:

Reverse proxy:
Nginx
#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by jonas154** [comment]
I've the same problem - get the following error message:

![WhatsApp Image 2021-01-01 at 11 20 40](https://user-images.githubusercontent.com/20500740/103437117-9d351480-4c23-11eb-895a-ff92ccfd3df0.jpeg)

**Comment by MarcS1975** [comment]
Yes. I think it's the same error or similar for me.

**Comment by AndyScherzinger** [comment]
Fixed for the upcoming release 3.14.2
